eng/versioning/version_client.txt

```diff
@@ -126,7 +126,7 @@ com.azure:azure-digitaltwins-core;1.3.19;1.4.0-beta.1
 com.azure:azure-e2e;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-health-insights-clinicalmatching;1.0.0-beta.1;1.0.0-beta.2
 com.azure:azure-health-insights-cancerprofiling;1.0.0-beta.1;1.0.0-beta.2
-com.azure:azure-health-insights-radiologyinsights;1.0.0-beta.1;1.0.0-beta.2
+com.azure:azure-health-insights-radiologyinsights;1.0.0-beta.1;1.0.0
```

Comment:
did you have a second beta of this package? If so, I would ask Java folks if it should stay in this file.

Reply:
Ah yes we might. I remember we had a second release shortly after the initial one. Not sure if it needs to be reflected in this file?

Reply:
There was no second beta for this package released. This got updated to the next preview-patch version by Azure SDK Bot after the beta1 release (https://github.com/Azure/azure-sdk-for-java/commit/d5f9c92711313253d8418eab379c323c817fa509).